Automattic/jetpack
# Pinterest: remove type hinting in function parameter

## Proposed changes:

For WordPress 6.2, WordPress updated its Requests library:
https://core.trac.wordpress.org/changeset/54997

This causes a Fatal error with the Pinterest block, since it expects an old version of the library.

Since we cannot use multiple type hints in PHP yet (gotta wait a bit more), let's remove the type hint for now. We can add it back when Jetpack will require WP 6.2

### Other information:


## Jetpack product discussion

* Primary issue: #27795

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Posts > Add New and add a Pinterest block. When adding a Pinterest URL, no Fatal error should appear in your logs.
* Install the WordPress Beta tester and switch to WP's Bleeding edge option.
* Update to WP 6.2 latest Beta in Dashboard > Updates
* Go to Posts > Add New and repeat your test. The Pinterest block should still trigger no fatal error.
